app/src/main/kotlin/com/bitwarden/authenticator/ui/authenticator/feature/itemlisting/util/VerificationCodeItemExtensions.kt

```diff
@@ -13,8 +13,13 @@ fun VerificationCodeItem.toDisplayItem(
     sharedVerificationCodesState: SharedVerificationCodesState,
 ) = VerificationCodeDisplayItem(
     id = id,
-    issuer = issuer,
-    label = accountName,
+    title = issuer ?: label ?: "--",
```

Comment:
This line of code seems wild, but I stole it from the main app:

https://github.com/bitwarden/android/blob/main/app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/itemlisting/VaultItemListingViewModel.kt#L1764